src/lib.rs

```diff
@@ -543,6 +544,31 @@ mod test {
         assert!(!MacAddr([1, 2, 3, 4, 5, 6]).is_unspecified());
     }
 
+    // Ensure that the `Display` and `Debug` traits are implemented on the `IpNetwork` struct
+    #[test]
+    fn check_display_impl_ip_network_ipv4() {
+        println!(
+            "{} {:?}",
+            IpNetwork::new(IpAddr::from(Ipv4Addr::new(1, 2, 3, 4)), 3),
+            IpNetwork::new(IpAddr::from(Ipv4Addr::new(255, 255, 255, 0)), 21)
+        );
+    }
+
+    #[test]
+    fn check_display_impl_ip_network_ipv6() {
+        println!(
+            "{} {:?}",
+            IpNetwork::new(
+                IpAddr::from(Ipv6Addr::new(0xffff, 0xaabb, 00, 0, 0, 0x000c, 11, 21)),
+                127
+            ),
+            IpNetwork::new(
+                IpAddr::from(Ipv6Addr::new(0xffcc, 0, 0, 0xffcc, 0, 0xffff, 0, 0xccaa)),
+                120
+            )
+        )
+    }
+
```

Comment:
Please add a test to ensure that supported platforms have at least one IP network for an interface. You can filter out using:

```rust
if !crate::IS_SUPPORTED_SYSTEM {
    return;
}
```

Reply:
Sure. Check added.